content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,51 +1,62 @@
 ====================
-Marketing Automation
+Marketing automation
 ====================
 
-The Odoo Marketing Automation app automates a variety of marketing tasks by combining specific
+The Odoo *Marketing Automation* app automates a variety of marketing tasks, by combining specific
 rules and filters to generate timed actions. Instead of manually having to build each stage of a
-campaign (such as a series of timed massmails), the Marketing Automation app allows marketers to
-build the entire campaign, and all of its stages, in one place on one dashboard.
+campaign (such as a series of timed massmails), the *Marketing Automation* app allows marketers to
+build the entire campaign, and all of its stages, in one place on single dashboard.
 
 Create a campaign
 =================
 
-To create a new automated marketing campaign, open the :guilabel:`Marketing Automation` app and
-click :guilabel:`Create`. On the :guilabel:`Campaign` page, there are the following smart buttons
-and fields:
+To create a new automated marketing campaign, open the :guilabel:`Marketing Automation` app,
```

Comment:
Use menuselection here because we are directing the user.

```suggestion
To create a new automated marketing campaign, open the :menuselection:`Marketing Automation` app,
```